test/await_cps_test.clj

```diff
@@ -205,6 +205,14 @@
                                        (b# [y# r# e#] (async r# e# (await a# y#)))]
                                  (await b# 1)))))))
 
+(deftest shadowing-symbols
+  ;; Here we shadow the `await` macro with a local function
+  (is (= 1 (:value (run-async timeout `(let [~'await (constantly 1)]
+                                         (~'await (fn [r# e#] (r# 2))))))))
```

Comment:
The assertion above seems to pass both with and without your patch applied if the namespace you're running the test from does not map `await` to `await-cps/await`. That's the case when loading a fresh REPL and kicking off tests from `user` namespace unless you `user => (require '[await-cps :refer [await]])`.

Any ideas how to make it reproducible regardless of environment?

Reply:
Yeah, this isn't too hard. If we bind `*ns*` in the test it will resolve properly. I'll push a commit to fix this soon.